modules/ccalib/src/ccalib.cpp

```diff
@@ -445,8 +445,11 @@ bool CustomPattern::findRtRANSAC(InputArray objectPoints, InputArray imagePoints
             OutputArray rvec, OutputArray tvec, bool useExtrinsicGuess, int iterationsCount,
             float reprojectionError, int minInliersCount, OutputArray inliers, int flags)
 {
+    float confidence_factor = minInliersCount / (imagePoints.total()/2);
```

Comment:
> imagePoints.total()/2

Why is half only?

---

Please use `checkVector()` instead of "2 coordinates trick".
Something like:

```
int npoints = std::max(1, objectPoints.getMat().checkVector(3));
```

Reply:
Fixed